sdk/go/auth.sdk.gen.go

```diff
@@ -4,6 +4,7 @@ package sdk
 
 import (
 	context "context"
+
```

Comment:
I need to get a step in our CI which runs `mage proto` and then makes sure nothing changes.

I think you might've just opened this file (and the other two) and saved and your editor reformatted them.
I am not sure if there is a nice way to avoid this (maybe some genproto rule to use `goimports` or something).

Reply:
oh I think I need to add an exception in the `mage fmt` task to ignore anything in the `sdk/go` subdir like I did for `rpc`. good call. will revert this part and add that as well